.travis.yml

```diff
@@ -11,7 +11,7 @@ deploy:
   provider: rubygems
```

Comment:
Let's remove this file - we need to migrate to Github Actions after merging this pr. Travis CI can't be relied on this days on free accounts.